README.md

```diff
@@ -12,22 +12,34 @@ Highly Available Tomcat 9 Image
        -> Update your application build.yaml to use this image for hosting your java application 
        -> Update your application deployment.yaml to add the rbac authorizations and roles required to run the tomcat in a cluster and don't forget to add the service account required to run your container under the spec.
 
-#### Note: To run with existing docker configuration, you will not be needed to make any changes to the docker folder
+#### Note: To run with existing docker configuration, you will not be need to make any changes to the docker folder
 
-# How to build image
-## In Openshift
+# Building & Deploying the Image
+## In Openshift (via developer CLI)
 ```
+We tag images using the name of the Jira Issue that caused the image to be built (the SUFFIX parameter).
```

Comment:
I thought we had agreed that image tags are based on pull-request number, not tickets.

Reply:
I don't think that's a very good developer workflow. Using this issue as an example:

I take ONETEAM-339.
I try to build an image. I supply the suffix.... I guess I just make something up?
It builds (or doesn't), I do my work, I keep making new builds.
Once it's ready to merge, I create a PR, and ... rebuild my image with suffix=-6, and every build I've done up to this point I... rename? Delete?

Reply:
The goal is that you should not build the image yourself, you MAY if you want, but the CI/CD server is the one of creating release candidates. Let's have more discussion on that.

https://guides.github.com/introduction/flow/

Reply:
We don't have a CI/CD server and we haven't implemented Github Flow. We shouldn't let the goals impede delivery of an MVP. https://blog.crisp.se/2016/01/25/henrikkniberg/making-sense-of-mvp 
That layout also doesn't specify when images should be built, since it's not equivalent to a deploy. It would be cool if opening a pull request built an image, but if that's the only way to build images we would generate a lot of pull request clutter. If you're suggesting images are only built post-merge, then it's not a practicable solution.

Reply:
My gut feeling is version tagging would be more understandable to the user. A CI/CD pipeline feels like overkill and there could still be manual steps like tagging the latest overall and the latest for a version. It'll be an interesting discussion tomorrow. I'll just leave this here.

https://stevelasker.blog/2018/03/01/docker-tagging-best-practices-for-tagging-and-versioning-docker-images/